airflow/jobs.py

```diff
@@ -459,39 +700,24 @@ def schedule_dag(self, dag):
                 session.commit()
                 return next_run
 
-    def process_dag(self, dag, queue):
+    def create_task_instances(self, dag, queue):
         """
         This method schedules a single DAG by looking at the latest
         run for each task and attempting to schedule the following run.
 
         As multiple schedulers may be running for redundancy, this
         function takes a lock on the DAG and timestamps the last run
-        in ``last_scheduler_run``.
+        in ``last_scheduler_run``. Note: multiple schedulers and locks are
+        not a fully implemented feature.
         """
         TI = models.TaskInstance
         DagModel = models.DagModel
         session = settings.Session()
 
-        # picklin'
-        pickle_id = None
-        if self.do_pickle and self.executor.__class__ not in (
-                executors.LocalExecutor, executors.SequentialExecutor):
-            pickle_id = dag.pickle(session).id
-
+        # Get a lock in the scheduler (not currently in use)
```

Comment:
This was reproducing existing lock functionality, but looks like that was removed.